src/app/auth/sign-up/sign-up.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, FormControl, Validators } from '@angular/forms';
+import { Store } from '@ngrx/store';
+
+import { AppState } from '../../store/app.state';
+import { register, logout } from '../store/auth.actions';
+
+import { emailValidator } from '../shared/validators/email.validator';
+import { passwordValidator } from '../shared/validators/password.validator';
+import { confirmPasswordValidator } from '../shared/validators/confirm-password.validator';
+import { ValidationService } from '../shared/validation.service';
+import { getAuthErrors } from '../store/auth.selector';
+
+@Component({
+  selector: 'sign-up',
```

Comment:
tslint considers this a warning:
`The selector should be prefixed by "app"`

Reply:
i have removed it from settings if it necessary i move it back